packages/framework/src/Form/Admin/Customer/CustomerFormType.php

```diff
@@ -57,7 +67,7 @@ public function configureOptions(OptionsResolver $resolver)
             ->setAllowedTypes('user', [User::class, 'null'])
             ->setAllowedTypes('domain_id', 'int')
             ->setDefaults([
-                'data_class' => CustomerData::class,
+                'empty_data' => $this->customerDataFactory->create(),
```

Comment:
Yes, unfortunatelly, it is intended. `CustomerData` must have parameters in constructor, because User, Billing and Delivery Data are created by factory (right after this commit).
Is it enought to write this sentence into CM?